icon-loader.js

```diff
@@ -1,12 +1,13 @@
 const VIEWBOX_REGEX = /viewBox="([^"]*)"/;
 const SVG_REGEX = /(<svg[^>]*>|<\/svg>)/g;
 const FILL_REGEX = /fill="[^"]*"/g;
+const WHITE_HEX_REGEX = /['"]#fff(?:fff)?['"]/gi;
 
 module.exports = function shopifyIconLoader(source) {
   this.cacheable(true);
 
   const finalSource = source.replace(FILL_REGEX, (fill) => {
-    return fill.includes('#FFF') ? 'fill="currentColor"' : '';
+    return fill.match(WHITE_HEX_REGEX) ? 'fill="currentColor"' : '';
```

Comment:
You do not need to use `match`, you should use `test` and just ditch the `g` flag on the regex.